src/jit/morph.cpp

```diff
@@ -3880,7 +3880,7 @@ void            Compiler::fgSetRngChkTarget(GenTreePtr  tree,
 {
     GenTreeBoundsChk* bndsChk = NULL;
 
-    if ((tree->gtOper == GT_ARR_BOUNDS_CHECK)
+    if (tree->gtOper == GT_ARR_BOUNDS_CHECK
```

Comment:
This change will also cause the code to be in violation of coding conventions for the FEATURE_SIMD case (which, BTW should be on by default).